README

```diff
@@ -224,23 +224,43 @@ Usage: cewl [OPTIONS] ... <url>
 	-c, --count: Show the count for each word found.
 	-v, --verbose: Verbose.
 	--debug: Extra debug information.
-      
+
 	Authentication
 	--auth_type: Digest or basic.
 	--auth_user: Authentication username.
 	--auth_pass: Authentication password.
-      
+
 	Proxy Support
 	--proxy_host: Proxy host.
 	--proxy_port: Proxy port, default 8080.
 	--proxy_username: Username for proxy, if required.
 	--proxy_password: Password for proxy, if required.
-      
+
 	Headers
 	--header, -H: In format name:value - can pass multiple.
-      
+
     <url>: The site to spider.
 
+Using Docker
+========
+
+Create container for CeWL
+```sh
+sudo docker build -t "greenmind/cewl:1" .
```

Comment:
Hey, I'm a little late but I think that having a Dockerfile would be cool for CeWL.

The commands in general need sudo, because if they didn't it would mean that the user is in the ``docker``group. Being in this group allows privilege escalation and gives access to root privileges, because Docker need root permissions do to its work with cgroups, networks, etc.
- More info here: https://fosterelli.co/privilege-escalation-via-docker.html
- And here : https://askubuntu.com/questions/477551/how-can-i-use-docker-without-sudo